src/pages/permission-management/role/list/Index.js

```diff
@@ -0,0 +1,204 @@
+import React, { PureComponent, Fragment } from 'react';
+import router from 'umi/router';
+import { connect } from 'dva';
+import {
+  List,
+  Card,
+  Row,
+  Col,
+  Radio,
+  Input,
+  InputNumber,
+  Progress,
+  Button,
+  Icon,
+  Dropdown,
+  Menu,
+  Avatar,
+  Modal,
+  Form,
+  DatePicker,
+  Select,
+  Badge,
+  Popconfirm,
+  message,
+  Divider,
+} from 'antd';
+import StandardTable from '@/components/StandardTable';
+import PageHeaderWrapper from '@/components/PageHeaderWrapper';
+import styles from '../../../List/TableList.less';
+const { MonthPicker, RangePicker, WeekPicker } = DatePicker;
+const FormItem = Form.Item;
+
+@connect(({ role, loading }) => ({
+  role,
+  loading: loading.models.role,
+}))
+@Form.create()
+class RoleList extends PureComponent {
+  state = {
```

Comment:
Codacy found an issue: [State initialization should be in a constructor](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)